docs/cheaha/open_ondemand/index.md

```diff
@@ -25,3 +25,40 @@ For a more complete description of how to select resources, go [here](ood_layout
 ## Debugging OOD Job Failures
 
 If your OOD job cards are disappearing after being allocated or during the job, see our [documentation](ood_layout.md#debugging-ood-job-failures) for instructions on how to retrieve the logs and submit a ticket to Research Computing support.
+
+## Choosing the right Jupyter Application
+
+Jupyter environments have revolutionized the way researchers, analysts, and developers interact with code. Whether you have workflows in Python, visualizing genomic data, running simulations or run any kind of computational workflow on our high-performance computing (HPC) cluster, you've likely encountered using [Jupyter Notebook](./ood_jupyter.md) and/or [JupyterLab](./ood_jupyterlab.md), we have detailed information for using both on Cheaha. Both applications are open-source web tools designed for creating and sharing documents that can include your code, equations, visualizations, and narrative text. Their main purpose is to offer an interactive environment for all phases of your data analysis life cycle including, data exploration, analysis, and visualization. But when should you choose one over the other on Cheaha to run your computational workflow?
+
+This article tries to help you decide, by comparing both Jupyter Apps, looking at their strengths and weaknesses, and how they measure up to your needs, especially on Cheaha.
+
+### What They Have in Common
+
+Before listing out some of the differences, the following are similarities they have in common;
```

Comment:
"similarities" and "in common" are redundant, stick with just one.

Since you reference "they have" here, you don't need to write it for any of the list items below. Alternatively, you can leave that part off and simply write "Below are similarities between X and Y".

End this with a full stop `.` character, instead of a semicolon `;` character.